Lib/fontbakery/profiles/universal.py

```diff
@@ -538,29 +538,31 @@ def com_google_fonts_check_whitespace_ink(ttFont):
 
 @check(
     id='com.google.fonts/check/required_tables',
-    conditions = ['is_ttf'],
```

Comment:
Hmmm... Interesting! This now makes me wonder whether ttFont as a condition should be explicitly declared on the vast majority of fontbakery checks...

Reply:
We'd need to review the portion of the checkrunner code that "magically" runs/skips checks based on file type. Maybe this specific usage of check conditions is not necessary due to that automation (as long as at least one of the known valid file types is not `None`), but I am not sure right now.

Reply:
> whether ttFont as a condition should be explicitly declared on the vast majority of fontbakery checks

Either that, or not specify conditions at all.
Since FB can process UFOs as well, I think the right solution is to be explicit.